### DISCUSSION TITLE
Error manually compiling React Suite (Sidebar)

### DISCUSSION THREAD

**Discussion by ChristianCaracach-velaONE** [rsuite/rsuite#3742] [Q&A] [answered]
Hi! In my company we use React Suite for our apps. In there, because we use also our own styles, we compile React Suite stlyes using lessc. We have a script that creates an index.less file (a list of .less files, one of them is importing the RSuite's styles) and then compiles everything using lessc.
```
---src/assets/general.less---

@import "../../../../node_modules/rsuite/styles/index.less";
// Here we import every other .less file
```

The thing is that we were using version "5.33.0" and we wanted to use RadioTile (needs "5.35.0" to work), so I decided to update the version (I've installed "5.59.0") . From that moment the Sidebar started to look weird. (We are using Nav wrapped into a Sidenav, copied from docs)
![image](https://github.com/rsuite/rsuite/assets/117648266/040cd829-f01f-45e1-873d-779487e77b01)
![image](https://github.com/rsuite/rsuite/assets/117648266/1ce4f515-cd64-4220-a00b-75922b3916d0)

(Note how expanded items now seems to be in a row-oriented flexbox)

Bottom line, it took a while to fix this bug, but we did it (just installed the old version and that worked for us, for now). But could it be a breaking-change version? I'm not an experienced developer in semantic versioning, but I thought that moving from 5.33.0 to 5.59.0 shouldn't bring any problem.
Questions are:
It is a breaking change in deed?
Are we doing something wrong when we compile the .less files?
 Are we even using React Suite in the way it is supposed to be used?
 
 Thanks

**Comment by simonguo** [comment, marked as answer]
Hi @ChristianCaracach-velaONE .

In version [`v5.58.0`](https://github.com/rsuite/rsuite/releases/tag/v5.58.0), a new feature was added to the Dropdown component, which caused a breaking change. This issue has been fixed in version [`v5.59.1`](https://github.com/rsuite/rsuite/releases/tag/v5.59.1). You can try upgrading rsuite to version `v5.59.1` to see if it resolves the issue.

**Reply by ChristianCaracach-velaONE** [reply]
Thanks! I'll try it.
